tests/kernel/mem_heap/mheap_api_concept/src/test_mheap_concept.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Intel Corporation
+ * Copyright (c) 2020 Intel Corporation
```

Comment:
Don't replace the year in copyright, but add to it. For example, `Copyright (c) 2016, 2020 Intel Corporation`.

Reply:
Understood, I didn't know that.